src/sketchup-stl/exporter.rb

```diff
@@ -51,27 +51,22 @@ def self.select_export_file
         filename
       end
 
-      def self.export(path, options = OPTIONS)
-        filemode = 'w'
-        if RUBY_VERSION.to_f > 1.8
-          filemode << ':ASCII-8BIT'
-        end
-        file = File.new(path , filemode)
-        if options['stl_format'] == STL_BINARY
-          file.binmode
-          @write_face = method(:write_face_binary)
-        else
-          @write_face = method(:write_face_ascii)
-        end
-        scale = scale_factor(options['export_units'])
-        write_header(file, model_name(), options['stl_format'])
-        if options['selection_only']
-          export_ents = Sketchup.active_model.selection
-        else
-          export_ents = Sketchup.active_model.active_entities
-        end
-        facet_count = find_faces(file, export_ents, 0, scale, Geom::Transformation.new)
-        write_footer(file, facet_count, model_name(), options['stl_format'])
+      def self.export(path, export_entities, options = OPTIONS)
+         filemode = 'w'
+         if RUBY_VERSION.to_f > 1.8
+            filemode << ':ASCII-8BIT'
+         end
+         file = File.new(path , filemode)
+         if options['stl_format'] == STL_BINARY
+            file.binmode
+            @write_face = method(:write_face_binary)
+         else
+            @write_face = method(:write_face_ascii)
+         end
+         scale = scale_factor(options['export_units'])
+         write_header(file, model_name(), options['stl_format'])
```

Comment:
I know - I just never liked the way a name in Ruby could either be a variable or method.  WIll comply.